servant/test/Servant/API/ContentTypesSpec.hs

```diff
@@ -219,16 +219,6 @@ spec = describe "Servant.API.ContentTypes" $ do
                 handleCTypeH (Proxy :: Proxy '[JSONorText]) "image/jpeg"
                     "foobar" `shouldBe` (Nothing :: Maybe (Either String Int))
 
-    -- aeson >= 0.9 decodes top-level strings
-    describe "eitherDecodeLenient" $ do
-
-        it "parses top-level strings" $ do
```

Comment:
It would be best if we could retain this test, ideally by having it use the `MimeUnrender JSON` instance that was modified. Just to make sure the docs are not mistaken.